### DISCUSSION TITLE
Community Discord Server

### DISCUSSION THREAD

**Discussion by awfixer** [zed-industries/zed#19469] [Show and tell]
Hello, my name is Austin, and I want to start this out by saying Zed is better than VSCode. 

I am not aiming for a sob story so I will keep this short. I had a stroke in February, and as a result of this will likely never be the developer I once was, so I have been looking for other ways to contribute to the community. 

The way I found that I can hopefully help this community is by helping with the community itself, in the form of a Discord Server. I can not seem to find one, so I went ahead and made one. 

I know it can be a challenge maintaining a large and Diverse Community, and I am fully prepared for that. I am experienced with moderation and management in community settings. 

There are some roles in the server like Zed Owner(s), Developer, and Contributer, and should this work I am willing to have one of the developers who works on my projects copy over the bot we have been working on that allows you to verify your GitHub as a contributing member and grants you the role. 

I have included a screenshot of the public channels and a link to the Discord Server below, it's me and Sapphire there at the moment, so it is kind of lonely. Hope to see you there! 

![Screenshot_20241019-124709.png](https://github.com/user-attachments/assets/f89775ac-afc1-4307-bd69-6052abf7d06b)

https://discord.gg/7hjnxTZsHG

**Comment by awfixer** [comment]
Just so everyone is aware, the server will continue to exist, but if you just join and leave it will always be lonely. I am still there and I will just politely wait

**Comment by jnptk** [comment]
> I can not seem to find one, ...

@awfixer there is a community discord server, you can find a permanent invite link here: [https://zed.dev/docs/conversations#discord-channel](https://zed.dev/docs/conversations#discord-channel) or in this discussion: #6923
